v1/website/data/users.js

```diff
@@ -274,6 +281,13 @@ module.exports = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: 'React Native Sensors',
+    image: '/image/users/react-native-sensors.svg',
```

Comment:
This should be `/img/`

Reply:
Awwwsh, I usually name my folders with the full name.